birdhouse/env.local.example

```diff
@@ -16,21 +19,20 @@
 export SSL_CERTIFICATE="/path/to/ssl/cert.pem"  # *absolute* path to the nginx ssl certificate, path and key bundle
 export PAVICS_FQDN="hostname.domainname" # Fully qualified domain name of this Pavics installation
 export DOC_URL="https://www.example.com/" # URL where /doc gets redirected
-export MAGPIE_SECRET=itzaseekrit
-export MAGPIE_ADMIN_USERNAME=admin
+export MAGPIE_SECRET="itzaseekrit"
```

Comment:
Just a wild idea, but instead of hardcoding the default value here and then manually ensuring all the `__DEFAULT__{var}` are matching the hardcoded value here, why don't we directly use `__DEFAULT__{var}` here?  Ex: `export MAGPIE_SECRET="${__DEFAULT__MAGPIE_SECRET}"`

This way we eliminate the need for the hardcoded value between `env.local.example` and `default.env` to be manually in-sync and we prevent other maintainers from forgetting to add matching `__DEFAULT__` var for future new variables because the pattern will immediately visible here.

Reply:
Sounds good.